isis/src/control/objs/ControlPoint/ControlPoint.cpp

```diff
@@ -254,6 +254,7 @@ namespace Isis {
     // notify parent network if we have one
     if (parentNetwork) {
       parentNetwork->measureAdded(measure);
+      parentNetwork->emitNewMeasure(measure);
```

Comment:
We don't need to emit new measure here, because measureAdded will emit that already